scripts/pylib/twister/twisterlib/harness.py

```diff
@@ -400,12 +400,8 @@ def generate_command(self):
 
         if handler.options.pytest_args:
             command.extend(handler.options.pytest_args)
-            if pytest_args_yaml:
-                logger.warning(f'The pytest_args ({handler.options.pytest_args}) specified '
-                               'in the command line will override the pytest_args defined '
-                               f'in the YAML file {pytest_args_yaml}')
-        else:
-            command.extend(pytest_args_yaml)
+
+        command.extend(pytest_args_yaml)
```

Comment:
so if the same parameter is provided in both command line and yaml the latter takes precedence, right ?

Reply:
Yes, if `argparse` argument is set up to be used only once and a user tries to use this argument multiple times, only the last occurrence will be considered